### ISSUE TITLE
Represent the pole figures in m.r.d.

### ISSUE THREAD

**Issue by lchoisez** [mtex-toolbox/mtex#75]
Hi,

I recently started using mtex (version mtex-4.0.23) and I am trying to represent some pole figures normalized in multiples of a random distribution.
I managed plotting great normalized pole figures by using the function plotPDF but the error given by the function plotDiff(pf,odf) can be high for some data.

Is it possible to normalize the pole figures in mrd without going through the ODF transformation to avoid the loss of information? I tried the function normalize(pf) but the obtained pole figures have a colorbar with negative data, so I suppose it is not mrd. 

My code is given below.

    %% Import Script for PoleFigure Data
    %
    % This script was automatically created by the import wizard. You should
    % run the whoole script or parts of it in order to import your data. There
    % is no problem in making any changes to this script.

    %% Specify Crystal and Specimen Symmetries

    % crystal symmetry
    CS = crystalSymmetry('2/m11', [5.2 5.2 5.36], [90,99,90]*degree, 'X||a', 'Y||b', 'Z||c*', 'color', 'light         blue');

    % specimen symmetry
    SS = specimenSymmetry('1');

    % plotting convention
    setMTEXpref('xAxisDirection','north');
    setMTEXpref('zAxisDirection','outOfPlane');

    %% Specify File Names
 
    % path to files
    pname = 'C:\Users\Hang Yu\Dropbox (MIT)\IPF_plot';

    % which files to be imported
    fname = {...
      [pname '\Stress_111.uxd'],...
      [pname '\Stress002.uxd'],...
      [pname '\Stress111.uxd'],...
      };

    %% Specify Miller Indice

    h = { ...
      Miller(-1, 1, 1,CS),...
      Miller(0,0,2,CS),...
      Miller(1,1,1,CS),...
      };

    %% Import the Data

    % create a Pole Figure variable containing the data
    pf_Stress = loadPoleFigure(fname,h,CS,SS,'interface','uxd');
    pf_back_Stress = correct(pf_Stress,'background',pf_Random);
    odf_Stress = calcODF(pf_back_Stress);
    plotPDF(odf_Stress,pf_back_Stress.h,'antipodal');
    annotate([xvector,yvector,zvector],'label',{'X','Y','Z'},'BackgroundColor','w');
    % plotDiff(pf_back_Stress,odf_Stress);

    % pf_norm_Stress = normalize(pf_back_Stress);
    % plot(pf_norm_Stress)

Pole figures from the function plotPDF
![pf_fromodf_stress](https://cloud.githubusercontent.com/assets/13455106/8831902/7bb98d3a-3073-11e5-95f0-fa875b16ec06.png)

Error from the transformation to ODF
![error_fromodf_stress](https://cloud.githubusercontent.com/assets/13455106/8831893/64dbb692-3073-11e5-8649-2efe19ae73f8.png)

Normalized pole figures with the function normalize
![pf_norm_stress](https://cloud.githubusercontent.com/assets/13455106/8831881/56cce350-3073-11e5-8676-72c2d4850be1.png)

**Comment by ralfHielscher** [comment]
Hi,

if you normalized pole figure show up negative values this mean that your data contain negative values. You can either set all negative data to zero and then perform normalization by
``` matlab
pf.intensities(pf.intensities < 0) = 0
pf_norm = normalize(pf);
```
or, and I think this is the better choice, spend some time investigating why you got negative values. Looking at you data I would suspect some systematic error.

Ralf.

**Comment by lchoisez** [comment]
Hi,

Yes I did not use the function 'correct' correctly, I used it to withdraw the background with a pole figure coming from a random sample instead of making a pole figure with only the background.

When using the 'uncorrected' pole figures, and normalizing them I get great pole figures!
But there are not as 'smooth' as the ones coming from plotPDF and I still have some loss of information using the transformation to ODF with plotPDF (there is an added inner circle in the (-111) pf) , is there a way to 'smooth' the normalized pole figures without plotPDF?

Normalized pole figures from the function normalized
![plot_pf_stress](https://cloud.githubusercontent.com/assets/13455106/8945305/498f2546-3554-11e5-8957-b0f40b8112e5.png)
Smooth pole figures from the function plotPDF
![pf_norm_stress](https://cloud.githubusercontent.com/assets/13455106/8945694/9e00e2ce-3557-11e5-8739-b36de9614226.png)
Loss of information plotted with the function plotDiff
![error_odftopf_norm_stress](https://cloud.githubusercontent.com/assets/13455106/8945788/95d7f082-3558-11e5-9580-e10c725534a5.png)
